main.go

```diff
@@ -214,7 +214,7 @@ func main() {
 	flag.StringVar(&fluentBitMemoryRequest, "fluent-bit-memory-request", "256Mi", "Memory request for fluent-bit")
 	flag.StringVar(&fluentBitImageVersion, "fluent-bit-image", fluentBitImage, "Image for fluent-bit")
 	flag.StringVar(&fluentBitExporterVersion, "fluent-bit-exporter-image", fluentBitExporterImage, "Image for exporting fluent bit filesystem usage")
-	flag.StringVar(&fluentBitPriorityClassName, "fluent-bit-priority-class-name", "", "Name of the priority class of fluent bit ")
+	flag.StringVar(&fluentBitPriorityClassName, "fluent-bit-priority-class-name", "telemetry-priority-class-high", "Name of the priority class of fluent bit ")
```

Comment:
Let's not make this a default to still allow running the operator for local development.